src/app/collective-rewards/components/AnnualBackersIncentives/AnnualBackersIncentives.tsx

```diff
@@ -0,0 +1,27 @@
+import { CommonComponentProps } from '@/components/commonProps'
+import { LoadingSpinner } from '@/components/LoadingSpinner/LoadingSpinner'
+import { Typography } from '@/components/TypographyNew/Typography'
+import { AnnualBackerIncentives } from '@/app/shared/components/AnnualBackersIncentives/AnnualBackerIncentives'
+import { Metric } from '@/components/Metric'
+
+interface AnnualBackersIncentivesProps extends CommonComponentProps {}
+
+export const AnnualBackersIncentives = ({ className }: AnnualBackersIncentivesProps) => {
```

Comment:
Not sure about this name?

Reply:
I agree, too many components with the same name make it difficult to read

Reply:
we renamed it to include the suffix loader